wagtail/tests/customuser/fixtures/test_explorable_pages.json

```diff
@@ -0,0 +1,112 @@
+[
```

Comment:
Probably for future PR, but is it really necessary for this test case to have its own users fixture?